### DISCUSSION TITLE
nokia g-2425g-a help

### DISCUSSION THREAD

**Discussion by DonuyThompson** [Anime4000/RTL960x#351] [Help]
nokia g-2425g-a
S/N: ALCLxxxxxx

Good morning, is it possible to change the serial number of this router SN?

**Comment by Anime4000** [comment]
this depend, my country, most stock ONU can change SN, your ONU likely can't change SN due to ISP request

**Reply by DonuyThompson** [reply]
> this depend, my country, most stock ONU can change SN, your ONU likely can't change SN due to ISP request

Send me the appropriate commands and I will try them.
